tests/dummy/app/templates/docs/buttons.md

```diff
@@ -44,3 +44,30 @@ Additionally there are many optional parameters for the underlying [UkButton](ht
 - `color`
 - `size`
 - `title`
+
+### TaskButton
+
+TaskButton uses the WorkItemButton as the foundation. Instead of passing a work item id, it taskes a task slug and graphql filters for the allWorkItems query and fetches the work item to be mutated.
+
+```hbs
+<TaskButton
+  @label="Lorem Ipsum"
+  @mutation="complete"
+  @taskSlug="some-task"
+  @filters="this.filters"
```

Comment:
```suggestion
  @filters={{array (hash myFilter="test")}}
```

Same as the last time. No one knows what `this.filters` is supposed to be

Reply:
Applied changes